.github/workflows/test-inputs.yml

```diff
@@ -429,3 +429,115 @@ jobs:
         uses: ./action/
         with:
           release-type: major
+
+  test-baseline-version:
+    # WIP: Help
+    name: Test baseline lookup with version from registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the test repository
+        uses: actions/checkout@v4
+        with:
+          repository: mgr0dzicki/cargo-semver-action-ref-slice
+          ref: major_change
+      - name: Checkout the action
+        uses: actions/checkout@v4
+        with:
+          path: action
+      - name: Run the action on ref_slice patch change
+        id: semver_check_step
+        uses: ./action/
+        with:
+          baseline-version: 1.2.1
+        continue-on-error: true
+      - name: Fail if the action has not returned any errors (but it should have)
+        if: steps.semver_check_step.outcome != 'failure'
+        run: |
+          echo "Error! The action should have failed because major_change is not compatible with crates.io 1.2.1"
+          exit 1
+
+  test-baseline-revision:
+    # Should pass because revision points to latest
+    # changes on major_change branch. If options are
+    # somehow ignored, this test should fail
+    name: Test baseline lookup with git revision
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the test repository
+        uses: actions/checkout@v4
+        with:
+          repository: mgr0dzicki/cargo-semver-action-ref-slice
+          ref: major_change
+      - name: Checkout the action
+        uses: actions/checkout@v4
+        with:
+          path: action
+      - name: debug
+        run: ls -la
+        shell: bash
```

Comment:
Is this still needed, or should I remove it before merging? It seems like a leftover from some debugging.

Reply:
Yep, leftover from the previous PR. Fixed.